backend/migrations/20230912181446_init.up.sql

```diff
@@ -0,0 +1,28 @@
+
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+ 
+CREATE TABLE attestation_requests (
+    id UUID PRIMARY KEY DEFAULT uuid_generate_v4()  NOT NULL,
+    approved BOOLEAN DEFAULT false  NOT NULL,
+    created_at TIMESTAMP DEFAULT now()  NOT NULL,
+    deleted_at TIMESTAMP,
+    updated_at TIMESTAMP,
+    ctype_hash VARCHAR(255)  NOT NULL,
+    credential jsonb  NOT NULL,
+    claimer VARCHAR(255)  NOT NULL,
+    revoked BOOLEAN DEFAULT false NOT NULL
+);
+
+
+CREATE OR REPLACE FUNCTION update_updated_at()
+RETURNS TRIGGER AS $$
+BEGIN
+    NEW.updated_at = NOW();
+    RETURN NEW;
+END;
+$$ LANGUAGE plpgsql;
+
+CREATE TRIGGER update
```

Comment:
unfortunately there is no CREATE TRIGGER IF NOT EXISTS (iirc) so this would need to be wrapped in a test

Reply:
I played a bit around and found this solution working for me: 

[e481678](https://github.com/KILTprotocol/attester/pull/1/commits/e481678e3283b8a9a1c8ba5c1ee0e133ef0ecc6a)